Casks/calmly-writer.rb

```diff
@@ -0,0 +1,23 @@
+cask "calmly-writer" do
+  version "2.0.25"
+  sha256 "92223bc82ca7af4718f41097edfa85da724f59b87b1a75d01a92db995325d48e"
+
+  url "https://www.calmlywriter.com/releases/arm64/Calmly%20Writer-#{version}.pkg"
```

Comment:
```suggestion

  if Hardware::CPU.intel?
    sha256 "269dffa6f93deb8116c865001e8862b0d97ff9246269ca29609aba5a0102a520"
    
    url "https://www.calmlywriter.com/releases/x64/Calmly%20Writer-#{version}.pkg"
  else
    sha256 "92223bc82ca7af4718f41097edfa85da724f59b87b1a75d01a92db995325d48e"

    url "https://www.calmlywriter.com/releases/arm64/Calmly%20Writer-#{version}.pkg"
  end

```

Currently, the cask is always using ARM64 version.
The checks are run on Intel 10.15, so probably causing install failure due to architecture difference.